### PR TITLE
adbtunnel command

### PR DESCRIPTION
**Pull request by jemoreira**

### CODE REVIEW

File: pkg/cli/adbtunnel.go [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+	"net"
+	"sync/atomic"
+
+	wclient "github.com/google/cloud-android-orchestration/pkg/webrtcclient"
```
Comment (ser-io) [inline pkg/cli/adbtunnel.go:22]: nit: please add a new line down below to separate the internal and third party import blocks

File: pkg/cli/adbtunnel.go [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+	"github.com/hashicorp/go-multierror"
+	"github.com/pion/webrtc/v3"
+	"github.com/spf13/cobra"
+)
+
+const hostFlag string = "host"
```
Comment (ser-io) [inline pkg/cli/adbtunnel.go:28]: nit: could this be moved to cli.go, it's going to be used by `cvd` subcommand as well?

File: pkg/cli/adbtunnel.go (hunk 1 above)
Comment (jemoreira) [reply pkg/cli/adbtunnel.go:22]: done

File: pkg/cli/adbtunnel.go (hunk 2 above)
Comment (jemoreira) [reply pkg/cli/adbtunnel.go]: done

File: pkg/cli/adbtunnel.go [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+type ADBForwarder struct {
+	cmd      *command
+	dc       *webrtc.DataChannel
+	host     string
+	device   string
+	listener net.Listener
+	conn     net.Conn
+	port     int
+	running  atomic.Bool
+}
```
Comment (ser-io) [inline pkg/cli/adbtunnel.go:111]: nit: wondering whether it's better to isolate the immutable properties like `cmd`, `host` and `device` into a sub structure, so it's more clear what changes and what not.

**Review by ser-io — APPROVED** [review]

File: pkg/cli/adbtunnel.go (hunk 3 above)
Comment (jemoreira) [reply pkg/cli/adbtunnel.go:111]: I don't like having those 3 in the ADBForwarder either, the only reason is so I can print logs. But I don't think the extra structure will help readability.

Grouped them together and added an empty line to separate them instead.

**Pull request merged**